plugins/discovery-azure-classic/build.gradle

```diff
@@ -63,6 +63,7 @@ File keystore = new File(project.buildDir, 'keystore/test-node.jks')
 
 // generate the keystore
 task createKey(type: LoggedExec) {
+  onlyIf { inFipsJvm == false }
```

Comment:
this task would fail as the JVM is setup for FIPS (BCFKS keystores, etc) but the dependency is not added when this is evaluated so the necessary classes are not available. We don't use the produced keystore eitherway as this can only work with a JKS keystore which is not available in FIPS - and this is why the `integTest` task is also disabled

Reply:
I'm not sure I follow. If integTest is disabled, what would try to run this task in FIPS mode?

Reply:
I guess it's because of `processTestResources.dependsOn(createKey)` below. I'd be happy to explore alternatives if you have suggestions!